Core/Blob/Data/BlobInitializerService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Lens.Core.Lib.Services;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Lens.Core.Blob.Data
+{
+    public class BlobInitializerService : BaseService<BlobInitializerService>, IProgramInitializer
```

Comment:
I see I mentioned the wrong class in my first comment. What I meant was that this specific BlobInitializerService does not add any additional logic towards the already existing DatabaseInitializerService in Lens.Core.Data.EF.Services. So to simplify and reduce the number of classes you also could use the DatabaseInitializerService instead of writing a copy